### DISCUSSION TITLE
Parallel extended and sigma-points smoothers in JAX

### DISCUSSION THREAD

**Discussion by AdrienCorenflos** [jax-ml/jax#5597] [Show and tell]
Hi,

I just wanted to share the arXiv release of our paper on Gaussian based parallel filters and smoothers for non linear state space models. It will appear this year as a conference paper at ICASSP.

The preprint can be found here https://arxiv.org/abs/2102.00514

And the code here https://github.com/EEA-sensors/parallel-non-linear-gaussian-smoothers

This publication follows this code: https://github.com/google/jax/discussions/5353

Adrien

**Comment by LB365** [comment]
Thank you Adrien, That's amazing :)

**Comment by mattjj** [comment]
Wow, awesome! Thanks for sharing this.
